static/js/layout.js

```diff
@@ -503,6 +504,23 @@ function formatAriaEthereumDate(elem) {
   }
 }
 
+function truncateTooltip() {
+  let nodes = $("[truncate-tooltip]")
+  nodes.each((_, node) => {
+    console.log(node)
+    let title = ""
+    if (node.scrollWidth > node.offsetWidth) {
+      title = node.attributes["truncate-tooltip"].value
+    }
+    if (node.attributes["title"]?.value != title) {
+      node.setAttribute("title", title)
+      if (title > 0) {
```

Comment:
Would this be preferred?

```suggestion
      if (title !== "") {
```

Reply:
thats better yes